mindsdb/integrations/handlers/prometheus_handler/README.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Prometheus
+sidebarTitle: Prometheus Handler
+---
+
+This documentation describes the intragration of MindsDB with [Prometheus](https://prometheus.io/), an open-source solution that allows you to collect different types of metrics from your host fleet in order to monitor their health, perfomance, usage, etc. 
+
+## Data Models 
+The data model used by Prometheus to store data is quite different that the relational model of SQL queries. Prometheus uses a timeseries model and exposes the stored data via PromQL. Translating any query from SQL to PromQL would be quite complex and no library was found that could serve that need. For the sake of providing some basic capability, we made some assumptions on how to map Prometheus' timeseries model to a relational model and we support specific types of queries which are simple to translate:
+
+1. Each Prometheus metric is mapped to a SQL table.
+2. Each Prometheus label is mapped to a SQL column. 
+2. We support SELECT queries with WHERE filters and AND conditions. 
+
+For example, consider the following PromQL query and its SQL mapping: 
+
+```
+http_requests_total{environment="development"}
+```
+
+```sql
+SELECT
```

Comment:
This represents how the query would be converted to be executed on MindsDB, right? If so, shall we put a data source name in front of `http_requests_total`? Like `prometheus_source.http_requests_total`?